src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/dto/AnalysisSistrResults.java

```diff
@@ -0,0 +1,42 @@
+package ca.corefacility.bioinformatics.irida.ria.web.analysis.dto;
+
+import java.util.Map;
+
+public class AnalysisSistrResults {
+	private String sampleName;
+	private Boolean parse_results_error;
+	private Map<String, Object> result;
```

Comment:
Don't like using `Object` in map, can you create another DTO for this?

Reply:
Done in 088174768afdeb42438c8ab4b3b797f26f25914a